tests-behat/bootstrap/Context.php

```diff
@@ -480,7 +478,7 @@ protected function jqueryWait($duration = 5000)
             }
         }
 
-        throw new \Exception('JQuery did not finished within a given time limit');
+        throw new \Exception('jQuery did not finished within a time limit');
```

Comment:
Should be "jQuery did not finish within the time limit"

Reply:
yes, thanks!